tests/stubs/devices/services/gatt.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package services
+
+import (
+	"log"
+
+	"github.com/paypal/gatt"
+)
+
+var (
+	attrGATTUUID           = gatt.UUID16(0x1801)
+	attrServiceChangedUUID = gatt.UUID16(0x2A05)
+)
+
+// NOTE: OS X provides GAP and GATT services, and they can't be customized.
+// For Linux/Embedded, however, this is something we want to fully control.
+func NewGattService() *gatt.Service {
+	s := gatt.NewService(attrGATTUUID)
+	s.AddCharacteristic(attrServiceChangedUUID).HandleNotifyFunc(
+		func(r gatt.Request, n gatt.Notifier) {
+			go func() {
+				//TODO: Indicate client when the services are changed
+				log.Printf("Creating a NewGattService")
```

Comment:
Could we maybe use glog to print the logs so as to maintain consistency with other files of the project?